Demo/Demo/SwiftUIComponents/PayPalWeb/PayPalWebStatusView.swift

```diff
@@ -0,0 +1,64 @@
+import SwiftUI
+
+struct PayPalWebStatusView: View {
```

Comment:
Similar Q - maybe more clear to name this something along the lines of "OrderID" status view. We could also add docstrings to the class-level to make things more clear. Also agree this should def be re-used across payment methods at some point

Reply:
What do we think of `PayPalWebOrderStatusView`?

Reply:
Same thing here, since it's technically a view of the `PayPalWebResult` we go with `PayPalWebResultView` on Android. A merchant typically would have their own `OrderCompleteView` style naming convention, but since our Demo apps are a glorified console log, I feel like it makes sense to couple the view to the underlying type it displays.

Reply:
We already have a `PayPalWebResultView` and since these views are conditionally displayed based on the status I think it'd be _more_ confusing to move the status view logic into here since it'd bloat this file quite a bit. Now if we simplify this view and display the same data across all status' ([as I mentioned here](https://github.com/paypal/paypal-ios/pull/223#discussion_r1412247314)) I think we could move this logic into our result view.

Reply:
Got it yeah, looking at ours we may need to clean up some of the naming. We go with `PayPalWebCheckoutResultView` to display the fields in a `PayPalWebCheckoutResult`. We do have some bloat on the Android side, probably a side effect of the UX. Another option would be to split everything up into screens to reduce the conditional logic, but that's a separate PR for sure.